apps/consulting/pages/nebari-services.tsx

```diff
@@ -372,11 +372,8 @@ export const NebariServicesPage: FC<TContainerProps> = ({
     <section className="bg-violet">
       <h2 className="sr-only">Testimonials</h2>
       <div
-        className="p-4 mx-auto sm:px-[3.4rem] xl:px-[11.5rem] max-w-layout"
-        style={{
-          background:
-            'url(/nebari-services/quansight-logo-violet-grayscale.svg) right bottom / 325px no-repeat',
```

Comment:
Moved this into Tailwind classnames so that I could more easily adjust the Quansight logo background with screen-width breakpoints